### PR TITLE
style: Made the FullScreen Button Transparent

### LINKED ISSUE #2234: Full screen icon does not show transparency
**Describe the bug**

The full screen button `activity_main_fullscreen_button` sets an alpha value of 0.6. However, the button is not transparent. 
**Expected behavior**

The button should have 60%~ transparency. 

**Steps to reproduce the behavior:**
1. Open full screen mode.
2. See that the full screen icon is either black or white (dark mode), without transparency. 

**Screenshots**

Screenshot_1595925259

### PR DESCRIPTION
**Pull request by s-ayush2903**
Close #2234, #2244 

Set the transparency of the button in java from xml

**Screenshots** 
ssss

### CODE REVIEW

**Comment by s-ayush2903** [comment]
This is regarding the screenshot posted above: the black bar being shown is due to the screenshot that was captured of the full screen and the current kiwix app didn't fill the screen till notch that's why that bar(empty space) is being shown.

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java [hunk 1]
```diff
@@ -1038,6 +1039,7 @@ protected void closeFullScreen() {
     toolbarContainer.setVisibility(View.VISIBLE);
     updateBottomToolbarVisibility();
     exitFullscreenButton.setVisibility(View.GONE);
+    exitFullscreenButton.getBackground().setAlpha(255);
```
Comment (Frans-Lukas) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java:1042]: Is this line fixing #2244 ? If not, I don't think it is necessary to change back to non-transparancy.

Comment (Frans-Lukas) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java:1042]: nvm, read your commit message. 👍

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java [hunk 2]
```diff
@@ -1024,6 +1024,7 @@ protected void openFullScreen() {
     toolbarContainer.setVisibility(View.GONE);
     bottomToolbar.setVisibility(View.GONE);
     exitFullscreenButton.setVisibility(View.VISIBLE);
+    exitFullscreenButton.getBackground().setAlpha(60);
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java:1027]: `60` isn't 60% of 255, ie what the old xml code was doing. The alpha apis are a bit terrible

**Comment by macgills** [comment]
@s-ayush2903 I won't require it for this PR but can you adhere to our [branch naming](https://github.com/kiwix/kiwix-android/blob/develop/CONTRIBUTING.md#branching) and [commit style](https://github.com/kiwix/kiwix-android/blob/develop/docs/commitstyle.md) on future work?

**Comment by s-ayush2903** [comment]
So, should i semantify the branch name and commit message now or close this pr(as you said that tou don't require this pr). I'm sorry for inconveniences

**Comment by s-ayush2903** [comment]
> @s-ayush2903 I won't require it for this PR but can you adhere to our [branch naming](https://github.com/kiwix/kiwix-android/blob/develop/CONTRIBUTING.md#branching) and [commit style](https://github.com/kiwix/kiwix-android/blob/develop/docs/commitstyle.md) on future work?

Definitely, I'll stick to them!

**Comment by macgills** [comment]
@s-ayush2903 just be aware of it on future prs/commits, right now we can merge this after you address my 1 comment

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java [hunk 3]
```diff
@@ -1024,6 +1024,7 @@ protected void openFullScreen() {
     toolbarContainer.setVisibility(View.GONE);
     bottomToolbar.setVisibility(View.GONE);
     exitFullscreenButton.setVisibility(View.VISIBLE);
+    exitFullscreenButton.getBackground().setAlpha(26);
```
Comment (s-ayush2903) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java:1027]: The exact 60% of 255 made the button hardly visible to an extent, so i increased the number by 10 and this looks better

Comment (macgills) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java:1027]: 60% of 255 is 153, I am not sure where 26+/-10 comes into this

**Pull request merged**